tests/python/utils.py

```diff
@@ -0,0 +1,67 @@
+# SPDX-FileCopyrightText: Copyright (c) 2023-present NVIDIA CORPORATION & AFFILIATES.
+# All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+# Owner(s): ["module: nvfuser"]
+
+import torch
+from torch.testing._internal.common_utils import TEST_WITH_ROCM
+from torch.testing._internal.jit_utils import RUN_CUDA
+
+# flake8: noqa
+from nvfuser import FusionDefinition, DataType
```

Comment:
`flake8` complains about `DataType` being unused in this file but is necessary to run captured fusion definition.

Reply:
let's leave this as a comment int he code.